classes/fields/time.php

```diff
@@ -169,6 +169,19 @@ public function is_empty( $value = null ) {
 		return empty( $value );
 	}
 
+	/**
+	 * {@inheritdoc}
+	 */
+	public function format_display( $options, $js = false ) {
+
+		if ( $js && 'custom' === pods_v( static::$type . '_type', $options, 'format' ) ) {
+			$format = $this->format_datetime( $options, $js );
+			return $this->convert_format( $format, array( 'source' => 'jquery_ui', 'type' => 'time' ) );
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 61077de299175146135635fefcb20192</sub>